client/network/src/service.rs

```diff
@@ -1025,6 +1043,27 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkService<B, H> {
 			.to_worker
 			.unbounded_send(ServiceToWorkerMsg::OwnBlockImported(hash, number));
 	}
+
+	// Utility function to extract `PeerId` from each `Multiaddr` for priority group updates.
+	fn parse_multiaddr(&self, peers: HashSet<Multiaddr>) -> Result<Vec<(PeerId, Multiaddr)>, String> {
```

Comment:
```suggestion
	fn split_multiaddr_and_peer_id(&self, peers: HashSet<Multiaddr>) -> Result<Vec<(PeerId, Multiaddr)>, String> {
```

I would find this more descriptive. What do you think?